src/Sylius/Bundle/CoreBundle/spec/DependencyInjection/Compiler/SitemapProviderCompilerPassSpec.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+ 
+namespace spec\Sylius\Bundle\CoreBundle\DependencyInjection\Compiler;
+ 
+use PhpSpec\ObjectBehavior;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Definition;
+use Symfony\Component\DependencyInjection\Reference;
+
+/**
+ * @author Arkadiusz Krakowiak <arkadiusz.krakowiak@lakion.com>
+ */
+class SitemapProviderCompilerPassSpec extends ObjectBehavior
```

Comment:
Specs for compiler pass are quite redundant and not really helpful, it is a better choice to write an integration test (https://github.com/matthiasnoback/SymfonyDependencyInjectionTest) :)